src/main/java/net/mcreator/ui/dialogs/SoundElementDialog.java

```diff
@@ -59,8 +61,9 @@ public static SoundElement soundDialog(MCreator mcreator, @Nullable SoundElement
 							.collect(Collectors.toList()));
 		}
 
-		JComboBox<String> soundCategory = new JComboBox<>(
-				new String[] { "master", "ambient", "player", "neutral", "hostile", "block", "record", "music" });
+		JComboBox<String> soundCategory = new JComboBox<>();

```

Comment:
As this JComboBox does not change, there is no need for updatecomboboxcontents, you can use https://docs.oracle.com/javase/7/docs/api/javax/swing/JComboBox.html#JComboBox(E[])